modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/object/relationship/ObjectRelationshipDDMFormFieldTemplateContextContributor.java

```diff
@@ -206,7 +206,7 @@ private String _getLabelKey(DDMFormField ddmFormField) {
 				objectDefinition.getTitleObjectFieldId());
 
 			if (objectField != null) {
-				return objectField.getName();
+				return _replaceDateLabelKey(objectField.getName());
```

Comment:
I've notice this bug happens too when we set Modified Date as title field.
You can to wrap both situations with this approach, since the fix is almost the same.
The ticket will be update to this.

Therefore, I think the best name to represent this method is `_parserLabelKey`